src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.GenericEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1/user")
+public class UserController {
+
+    @PostMapping(value = "createUser")
+    public GenericEntity createUser() {
+        return new GenericEntity("user_1");
+    }
+
+    @PutMapping(value = "blockedUser")
+    public GenericEntity blockedUser(Long userId) {
+        return new GenericEntity("user_blocked");
+    }
+
+    @PostMapping(value = "createUserKey")
+    public void createUserKey(Long userKey) {
```

Comment:
Ta metoda powinna zwracać jakiś klucz (może to być `String`) i przyjmować dwa argumenty login i hasło (albo jedną klasę mającą te argumenty)

Reply:
No właśnie niepotrzebnie robiłem rebase innych branchy. No ale chciałem potestować. No a przy commicie to nie wiem jak to poszło. Wydaje mi się, że zaznaczam tylko pliki, które chce aby się wysłały.
Co do metody createUserKey(), dokonuję zmian według twoich zaleceń.